decidim-collaborative_texts/app/views/decidim/collaborative_texts/admin/documents/edit.html.erb

```diff
@@ -7,7 +7,40 @@
 <div class="item__edit item__edit-1col">
   <div class="item__edit-form">
     <%= decidim_form_for(@form, url: document_path(document), html: { class: "form-defaults form edit_document" }) do |f| %>
-      <%= render partial: "form", object: f %>
+      <%= render partial: "form", object: f, locals: { has_suggestions: document.has_suggestions? } %>
+
+      <% if document.draft? %>
```

Comment:
maybe we could extract 2 partials here ?
```
if draft?
  render partial
else 
  render other partial
end 
```

Reply:
yep, good idea